LICENSE.txt

```diff
@@ -1,5 +1,5 @@
 Copyright (c) 2011 Lukas Lalinsky
-Copyright (c) 2011, 2012 MetaBrainz Foundation Inc.
```

Comment:
Ah, thanks for pointing that out. I was trying to update old ones. Some are inconsistent: one or multiple years listed, different versions of the text (short, long).

Should I just append `2015` to these?